src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +36,20 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getTypeExtensions()
+    {
+        $typeExtensions = array();
+
+        if (interface_exists(ValidatorInterface::class)) {
+            $validator = $this->getMockBuilder(ValidatorInterface::class)->getMock();
+            $validator->expects($this->any())->method('validate')->will($this->returnValue(array()));
```

Comment:
Then only this line should be added in the proper validator extension base type, we don't need to add the extension overhead to all base tests.